.github/actions/deploy_to_kubernetes/action.yml

```diff
@@ -0,0 +1,31 @@
+name: k8s_deploy
+description: Deploy to Kubernetes
+inputs:
+  TARGET:
+    description: Target environment
+    required: false
+    default: 'beta'
```

Comment:
does this have to be beta?

Reply:
I'm not quite sure @ali-hosseini-deriv. I copied the details from the `circleci` configurations, and this was the default. Should I change it?

Reply:
we are not using TARGET anywhere in this action. Do we need it?

Reply:
I'll remove it @balakrishna-deriv, @ali-hosseini-deriv